birdhouse/birdhouse-compose.sh

```diff
@@ -100,15 +128,10 @@ else
   log DEBUG "Skipping template files update (\"$1\" not \"up\", \"restart\", or forced by BIRDHOUSE_COMPOSE_TEMPLATE_FORCE)"
 fi
 
-SHELL_EXEC_FLAGS=
-if [ "${BIRDHOUSE_LOG_LEVEL}" = "DEBUG" ]; then
-  SHELL_EXEC_FLAGS=-x
-fi
-
 create_compose_conf_list # this sets COMPOSE_CONF_LIST
 log INFO "Displaying resolved compose configurations:"
 log INFO "COMPOSE_CONF_LIST="
-log INFO ${COMPOSE_CONF_LIST} | tr ' ' '\n' | grep -v '^-f'
+log INFO "$(echo ${COMPOSE_CONF_LIST} | tr ' ' '\n' | grep -v '^-f')"
```

Comment:
Should `${COMPOSE_CONF_LIST}` be in quotes as well, to avoid possible issues like a `>` or `|` character making its way there somehow?